man/systemd-repart.xml

```diff
@@ -40,16 +40,17 @@
     <citerefentry><refentrytitle>repart.d</refentrytitle><manvolnum>5</manvolnum></citerefentry>.
     </para>
 
-    <para>If invoked with no arguments, it operates on the block device backing the root file system partition
-    of the OS, thus growing and adding partitions of the booted OS image itself. When called in the initial
-    RAM disk it operates on the block device backing <filename>/sysroot/</filename> instead, i.e. on the
-    block device the system will soon transition into. The <filename>systemd-repart.service</filename>
-    service is generally run at boot in the initial RAM disk, in order to augment the partition table of the
-    OS before its partitions are mounted. <command>systemd-repart</command> (mostly) operates in a purely
-    incremental mode: it only grows existing and adds new partitions; it does not shrink, delete or move
-    existing partitions. The service is intended to be run on every boot, but when it detects that the
-    partition table already matches the installed <filename>repart.d/*.conf</filename> configuration
-    files, it executes no operation.</para>
+    <para>If invoked with no arguments, it operates on the block device backing the root file system
+    partition of the OS, thus growing and adding partitions of the booted OS image itself (unless
+    <varname>--image=</varname> is used in which case it will operate on the specified image file by
+    default). When called in the initial RAM disk it operates on the block device backing
```

Comment:
"by default" — hmm, I don't get this. I'd think that if `--image=` is specified, it'll operate on that image, period.

"When called in the initial RAM disk it operates" → "When invoked in the initrd, operates"
(It's not really "in the disk"... We say "in the initrd" everywhere, let's treat this as an idiomatic expression.)